composables/playlist.ts

```diff
@@ -35,3 +35,11 @@ export function useCuratedPlaylists() {
     useLazyAsyncData("playlists", () => new PlaylistApi().playlistGet()),
   );
 }
+
+export function useFeaturedPlaylists() {
+  return reactiveApi(
+    useLazyAsyncData("playlists-featured", () =>
```

Comment:
Just hope nobody names his playlist `featured` or we'll get in conflict with the caching of the function in line 11... maybe best to use a different caching-id there...

Reply:
it uses id, which is an integer. Also one uses singular whereas the other plural "playlist".